CHANGELOG

```diff
@@ -1,6 +1,10 @@
 devel
 -----
 
+* The General Graph document API is now persistent with the document API in its errormessages.
+  When attempting to create / modify edges pointing to non existing vertex collections
+  HTTP 400 is returned instead of 404.
```

Comment:
ah, it is changed back again?
I just had to adjust drivers for 3.5 because the return code got changed there. Now we are changing it back again...